native/src/components/MapView.tsx

```diff
@@ -37,126 +37,126 @@ const featureLayerId = 'point'
 
 // Has to be set even if we use map libre
 MapboxGL.setAccessToken(mapConfig.accessToken)
-const MapView = ({
-  boundingBox,
-  featureCollection,
-  selectedFeature,
-  fabPosition,
-  onRequestLocationPermission,
-  locationPermissionGranted,
-  selectPoiFeature,
-  setSheetSnapPointIndex
-}: MapViewPropsType): ReactElement => {
-  const [followUserLocation, setFollowUserLocation] = useState<boolean>(false)
-  const mapRef = React.useRef<MapboxGL.MapView | null>(null)
-  const cameraRef = React.useRef<MapboxGL.Camera | null>(null)
-  const theme = useTheme()
-
-  const layerProps: SymbolLayerProps = {
-    id: featureLayerId,
-    style: {
-      symbolPlacement: 'point',
-      symbolZOrder: 'source',
-      iconAllowOverlap: true,
-      iconSize: mapMarker.iconSize,
-      iconIgnorePlacement: true,
-      iconImage: [
-        'case',
-        ['==', ['get', 'id'], selectedFeature?.properties.id ?? -1],
-        mapMarker.symbolActive,
-        ['get', 'symbol']
-      ],
-      textField: ['get', 'title'],
-      textFont: ['Roboto Regular'],
-      textOffset: [0, textOffsetY],
-      textAnchor: 'top',
-      textSize: 12
+const MapView = React.forwardRef(
```

Comment:
:/ Seems like I have to read up on these forward refs because currently I don't get what it is doing and why it is needed...

Reply:
React forwardRef is a method that allows parent components pass down (i.e., “forward”) refs to their children.